src/style.css

```diff
@@ -1 +1,80 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable color-hex-case */
+/* stylelint-disable no-missing-end-of-source-newline */
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
add these styles only for needed selectors
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/3021993e-6ad7-4bc2-893a-3ad375c656b8)